integration/app/test-demo/src/pages/RosterTestApp.tsx

```diff
@@ -0,0 +1,58 @@
+import { MeetingProvider, useAttendeeStatus, useRosterState } from 'amazon-chime-sdk-component-library-react';
+import { LogLevel } from 'amazon-chime-sdk-js';
+import React, { FC } from 'react';
+import MeetingControls from '../containers/MeetingControls';
+import MeetingForm from '../containers/MeetingForm';
+import MeetingRoster from '../containers/MeetingRoster';
+
+export const RosterTestApp: FC = () => {
+  const config = {
+    logLevel: LogLevel.INFO,
+  };
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
+      <MeetingProvider {...config}>
+        <h3 data-testid="app-name">Roster Test</h3>
+        <MeetingForm />
+        <Meeting />
+      </MeetingProvider>
+    </div>
+  )
+};
+
+const Meeting: FC = () => {
+  return (
+    <>
+      <MeetingControls />
+      <div style={{ marginTop: '2rem', height: '40rem', display: 'flex', flexDirection: 'row' }}>
+        <MeetingRoster />
+        <HookStates />
+      </div>
+    </>
+  )
+};
+
+const HookStates: FC = () => {
+  const { roster } = useRosterState();
+
+  const states = Object.entries(roster).map(([attendeeId, attendee]) =>
+    <AttendeeState key={attendeeId} attendeeId={attendeeId} attendee={attendee} />
+  );
+
+  return (<><pre>{states}</pre></>);
+};
+
+const AttendeeState: FC<{ attendeeId: string, attendee: {} }> = ({ attendeeId, attendee }) => {
```

Comment:
Can we create an interface or `any` instead of `{}`?
I saw the type of Attendee is string according to your next file?

Reply:
Checked the API, the type is `RosterAttendeeType`. Updated.